pandas/tests/indexes/test_setops.py

```diff
@@ -195,7 +208,14 @@ def test_union_base(self, index):
         first = index[3:]
         second = index[:5]
         everything = index
-        union = first.union(second)
+
+        warn = None
+        if is_dtype_equal(index.dtype, "boolean") and index.isna().any():
+            warn = RuntimeWarning
```

Comment:
This might be no longer be needed now? First, the boolean index (defined in conftest?) doesn't actually have nulls at the moment (you might have removed them at some point?)

Second, also if I create a boolean index with missing values, I can't reproduce it triggering such warning:

```
In [5]: first = pd.Index([True, False, None]*2, dtype="boolean")

In [6]: second = pd.Index([False, True, None]*2, dtype="boolean")

In [7]: first.union(second)
Out[7]: Index([False, False, True, True, <NA>, <NA>], dtype='boolean')
```

Reply:
> doesn't actually have nulls at the moment (you might have removed them at some point?)

yep!  good catch, will update.